tests/cython/README.md

```diff
@@ -13,7 +13,7 @@ apt-get install python-nose
 You may need to set up the PYTHON path:
 export PYTHONPATH=${PROJECT_BINARY_DIR}/opencog/cython
 or, if installed:
-export PYTHONPATH=/usr/local/lib/python3.5/dist-packages/opencog:${PYTHON}
+export PYTHONPATH=/usr/local/lib/python3/dist-packages/opencog:${PYTHON}
```

Comment:
I have my opencog install in `/usr/local/lib/python{3.5,3.6}/site-packages/opencog` and not specifying a minor version will cause all sorts of breakages.

(which isn't to say one of our cmake files installs stuff to /usr/local/lib/python3/dist-packages/opencog - but if that's the case I think we should fix it to avoid pain in future)

I saw your comment [here](https://github.com/opencog/atomspace/issues/1782#issuecomment-500492476), so looks like it shouldn't be too hard to get the correct path despite the distutils bug.

Reply:
Ok - the reason I was getting things installed to '/usr/local/lib/python{3.5,3.6}/site-packages' instead of '/usr/local/lib/python3/dist-packages' like everyone else is that I was running the build in a virtualenv (while testing cython compatibility).